arch/risc-v/include/irq.h

```diff
@@ -562,9 +570,15 @@ struct xcptcontext
 #endif
 #endif
 
-  /* Register save area */
+  /* Integer register save area */
 
   uintptr_t *regs;
+
+  /* FPU register save area */
+
+#if defined(CONFIG_ARCH_FPU) && defined(CONFIG_ARCH_LAZYFPU)
+  uintptr_t fregs[FPU_XCPT_SIZE];
```

Comment:
Just noticed that this needs to be fixed too, this now reserves 4-8 times more memory for the FPU regs than is needed. Should be FPU_XCPT_REGS